apps/src/StudioApp.js

```
@@ -1629,7 +1629,9 @@ StudioApp.prototype.runButtonClickWrapper = function (callback) {
  */
 StudioApp.prototype.configureDom = function (config) {
   var container = document.getElementById(config.containerId);
-  container.innerHTML = config.html;
+  if (typeof config.html !== 'undefined') {
+    container.innerHTML = config.html;
+  }
```

Why is it safe to silently ignore an undefined config.html?